File: src/main/java/hits/internship/NotificationService/model/kafka/NewComment.java
package hits.internship.NotificationService.model.kafka;

import hits.internship.NotificationService.model.enumeration.CommentType;
import lombok.Data;
import lombok.EqualsAndHashCode;

import java.util.UUID;

@EqualsAndHashCode(callSuper = true)
@Data
public class NewComment extends KafkaMessageRequest{
    CommentType commentType;
    String message;
    String fullName;
    UUID typeId;
}

File: dotnet/NotificationModule.Kafka/Messages/NewComment.cs
using NotificationModule.Domain.Enums;

namespace NotificationModule.Kafka.Messages;

public class NewComment : KafkaMessageRequest
{
    public CommentType CommentType { get; set; }
    public string Message { get; set; }
    public string FullName { get; set; }
    public Guid Typeid { get; set; }
}